zulipterminal/api_types.py

```diff
@@ -83,13 +83,15 @@ class PrivateComposition(TypedDict):
     type: DirectMessageString
     content: str
     to: List[int]  # User ids
+    read_by_sender: bool  # New in ZFL 58, Zulip 4.0
```

Comment:
These new comments are incorrect.

Reply:
To clarify, the use of a comment here is fine, and in accordance with the rest of the file. However, this field was not introduced at this zulip version / feature level.